code/game/objects/items/devices/sound_synth.dm

```diff
@@ -0,0 +1,64 @@
+/obj/item/device/soundsynth
```

Comment:
why is this a device? does that subtype even exist?

Reply:
Nope, can't believe I didn't realize stuff like multitools weren't devices despite being under the device folder, it's fixed now!